pkg/operator/ceph/csi/csidriver.go

```diff
@@ -70,7 +78,8 @@ func (d v1CsiDriver) createCSIDriverInfo(ctx context.Context, clientset kubernet
 
 	// As FSGroupPolicy and AttachRequired fields are immutable, should be set only during create time.
 	// if the request is to change the FSGroupPolicy or AttachRequired, we are deleting the CSIDriver object and creating it.
-	if (driver.Spec.FSGroupPolicy != nil && csiDriver.Spec.FSGroupPolicy != nil && *driver.Spec.FSGroupPolicy != *csiDriver.Spec.FSGroupPolicy) || *driver.Spec.AttachRequired != *csiDriver.Spec.AttachRequired {
+	if (driver.Spec.FSGroupPolicy != nil && csiDriver.Spec.FSGroupPolicy != nil && *driver.Spec.FSGroupPolicy != *csiDriver.Spec.FSGroupPolicy) ||
+		(driver.Spec.SELinuxMount != nil && csiDriver.Spec.SELinuxMount != nil && driver.Spec.SELinuxMount != csiDriver.Spec.SELinuxMount) {
```

Comment:
Not required here as SelinxMount flag is [mutable](https://github.com/kubernetes/api/blob/366128ee4928cb288c105f7529e896226554dc99/storage/v1/types.go#L397-L416), can you please check

Reply:
thanks, modified !